packages/react-components/src/components/Attachment/AttachmentCard.tsx

```diff
@@ -136,20 +136,26 @@ export const _AttachmentCard = (props: _AttachmentCardProps): JSX.Element => {
                   target: '#attachment-' + attachment.id
                 }}
               >
-                <Text className={attachmentCardStyles.title} title={attachment.name}>
-                  {attachment.name}
-                </Text>
+                <div className={attachmentCardStyles.fileNameLabel} tabIndex={0}>
```

Comment:
Just wondering if we need `tabIndex={0}` here or we can access the `Text`'s aria-label without it?

Reply:
yes, without this, text won't be accessible by keyboard

Reply:
Let's have a quick discussion with Aman on this, as we shouldn't make non-interactive elements keyboard navigable.

Reply:
As discussed let's remove this tabindex{0} for now and merge this PR as it addressed two bugs.

For the title portion/bug, we can talk to aman and create separate PR if needed. Thanks!

Reply:
had a chat with patrick offline, I will remove tabIndex={0} so title will no longer be focusable. If our call with A11y team determines that title should still be focusable even though it's a non interactive element then I will have a separate PR to fix it

Reply:
https://github.com/Azure/communication-ui-library/assets/109105353/36c20261-7150-494c-b99c-0bd41362d7ec